datajs/src/file-base.js

```diff
@@ -128,8 +130,6 @@ export class FileInterface extends File {
   stream({ size } = {}) {
     size = size === -1 ? this.size : size || 0
     return toNodeStream(this.descriptor.stream().getReader(), size)
```

Comment:
This should use the new implemented function, we don;t need `toNodeStream()` function.